cost-analyzer/values.yaml

```diff
@@ -205,6 +205,50 @@ kubecost:
     #  cpu: "100m"
     #  memory: "256Mi"
 
+# Kubecost Metrics deploys a separate pod which will emit kubernetes specific metrics required 
+# by the cost-model. This pod is designed to remain active and decoupled from the cost-model itself. 
+# However, disabling this service/pod deployment will flag the cost-model to emit the metrics instead. 
+kubecostMetrics: 
+  # emitPodAnnotations: false 
+  # emitNamespaceAnnotations: false 
+  # emitKsmV1Metrics: true
```

Comment:
Are these intentionally left here?

Reply:
Yes, and they're still very relevant for the regular cost-model, exporter, and agent.